river/IdleInhibitorManager.zig

```diff
@@ -0,0 +1,64 @@
+const Self = @This();
+
+const std = @import("std");
+const wlr = @import("wlroots");
+const wl = @import("wayland").server.wl;
+
+const server = &@import("main.zig").server;
+const util = @import("util.zig");
+
+const View = @import("View.zig");
+const IdleInhibitor = @import("IdleInhibitor.zig");
+
+idle_inhibit_manager: *wlr.IdleInhibitManagerV1,
+new_idle_inhibitor: wl.Listener(*wlr.IdleInhibitorV1),
+inhibitors: std.TailQueue(IdleInhibitor) = .{},
+idle: *wlr.Idle,
+
+pub fn init(self: *Self, idle: *wlr.Idle) !void {
+    self.idle_inhibit_manager = try wlr.IdleInhibitManagerV1.create(server.wl_server);
+    self.idle = idle;
+    self.new_idle_inhibitor.setNotify(handleNewIdleInhibitor);
+    self.idle_inhibit_manager.events.new_inhibitor.add(&self.new_idle_inhibitor);
+}
+
+pub fn deinit(self: *Self) void {
+    while (self.inhibitors.pop()) |inhibitor| {
+        util.gpa.destroy(inhibitor);
+    }
+}
+
+pub fn idleInhibitCheckActive(self: *Self) void {
+    var inhibited = false;
+    var it = self.inhibitors.first;
+    while (it) |node| : (it = node.next) {
+        var view = View.fromWlrSurface(node.data.inhibitor.surface);
+        if (view) |v| {
```

Comment:
I'd write this like so, what do you think?
```suggestion
        if (View.fromWlrSurface(node.data.inhibitor.surface)) |view| {
```

Reply:
Changed it in [93a95b4](https://github.com/riverwm/river/pull/568/commits/93a95b4c6a7448e463697e00fce3ba24db51337b)